### PR TITLE
Completed string-manipulation-1

### PR DESCRIPTION
**Pull request by jeff-cruz**
Screen Shot 2022-03-25 at 11 41 03 AM

### CODE REVIEW

File: string-manipulation-1/src/get-words.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+function getWords(string) {
+  var array = [];
+  var word = string.split(' ');
+  for (var i = 0; i < word.length; i++) {
+    if (string === '') {
+      array.push();
+    } else {
+      array.push(word[i]);
+    }
+  }
```
Comment (codingcodymiller) [inline string-manipulation-1/src/get-words.js:11]: Did you try logging out the value of `word` to see what it is? This loop isn't necessary

File: string-manipulation-1/src/is-vowel.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/* exported isVowel */
+function isVowel(character) {
+  if (character === 'a' || character === 'e' || character === 'i' || character === 'o' || character === 'u' || character === 'A' || character === 'E' || character === 'I' || character === 'O' || character === 'U') {
```
Comment (codingcodymiller) [inline string-manipulation-1/src/is-vowel.js:3]: If you lowercase or uppercase the letter before the conditional, you can cut the amount of items to check by half 🤓

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work on the pseudocode!!

File: string-manipulation-1/src/is-vowel.js (hunk 2 above)
Comment (jeff-cruz) [reply string-manipulation-1/src/is-vowel.js:3]: I knew this code didn't seem right haha

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**